GoogleForCreators/web-stories-wp
# Dashboard: Integrate new theme values

## Summary

Lays groundwork to progressively add new design system components to dashboard while maintaining separation of reusable components from Figma and not breaking anything existing in dashboard.

## Relevant Technical Choices

- The only thing I did here was import the new theme from the design system into the `ThemeProvider` and set the internal theme that is within the dashboard to be the `internalTheme` nested inside the overall `theme` so that the new values are present and structured appropriately and it's super easy to tell what hasn't been updated. 
- All of the updates in here are just drilling 1 level deeper in to the newly dubbed "internalTheme".
- I figure doing it this way we can iteratively update to the new dashboard designs using components from the shared design system component library without breaking anything.

## To-do
Everything, ha! This is purely groundwork. 

## User-facing changes

Nothing should change. 

## Testing Instructions

- Run storybook and click on things in the dashboard section, see that nothing breaks due to theme (there's some provider stuff that has a few demos not working, but those will be fixed later) 
- Load the dashboard locally and click around, open a dialog, fail an API request to see a toast see that nothing breaks. 
(It'll be super obvious if something is awry because the page won't load) 

---



Addresses #5008
